swarm/network/simulation/simulation_test.go

```diff
@@ -205,3 +205,34 @@ func (t *noopService) Start(server *p2p.Server) error {
 func (t *noopService) Stop() error {
 	return nil
 }
+
+// a helper function for most basic noop service
+// of a different type then noopService to test
+// multiple services on one node.
+func noopService2Func(ctx *adapters.ServiceContext, b *sync.Map) (node.Service, func(), error) {
+	return newNoopService2(), nil, nil
+}
+
+// noopService2 is the service that does not do anything
```

Comment:
why not just embed `noopService`? no need to define functions then

Reply:
Nice catch, changed.

Reply:
yes, agree, nicer.